code/utils/HeapAllocator.cpp

```diff
@@ -177,6 +177,7 @@ bool HeapAllocator::check_connected_range(const MemoryRange& left, const MemoryR
 	return left.offset + left.size == right.offset;
 }
 void HeapAllocator::checkRangesMerged() {
+#ifndef NDEBUG
```

Comment:
```suggestion
// This is a debug only function because its values are only used in debug and linters will get tripped up otherwise.
#ifndef NDEBUG
```

Reply:
I think it'd be good to mark this with a comment so that we don't have to look through commit history to know what's going on here.  You can write your own version if you want, of course.

Reply:
Good idea.  After a bit of consideration I think this version works. 👍